charts/index.yaml

```diff
@@ -3,19 +3,19 @@ entries:
   azure-service-operator:
   - apiVersion: v2
     appVersion: 0.1.0
-    created: "2020-07-08T10:09:00.884245-06:00"
+    created: "2020-07-16T21:47:46.9704846-07:00"
     dependencies:
-    - condition: azureUseMI
+    - condition: installAadPodIdentity
       name: aad-pod-identity
       repository: https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts
       version: 1.5.5
     description: Deploy components and dependencies of azure-service-operator
-    digest: 7772dcf19b531eced14cec948a8d2705c19303d5c210ff2f0b52784931baeb19
+    digest: d494f8aa0dd7507bf3fbdef78aa8c5bee5adf8069d5a6a6748a22d463f88d4eb
     home: https://github.com/Azure/azure-service-operator
     name: azure-service-operator
     sources:
     - https://github.com/Azure/azure-service-operator
     urls:
-    - azure-service-operator-0.1.0.tgz
-    version: 0.1.0
-generated: "2020-07-08T10:09:00.877634-06:00"
+    - azure-service-operator-0.1.1.tgz
+    version: 0.1.1
```

Comment:
The CI/CD around the helm chart is a bit simple right now. It doesn't do anything with versioning. As such, the tar being used in the pipeline has the 10.1.0` version hardcoded. 

If we want to adhere to semver in the helm chart we will need to ensure that our current usage of MCR allows storage of multiple versions. Also we need to decide if it's worth the trouble. Helm won't currently allow us to update or delete CRDs so the value proposition is dubious.

Reply:
Ah, sorry. I'm used to the stable repo's CI/CD where they lint the chart and reject it without a version bump. Want me to revert the version number and repack the chart?